app/Main.hs

```diff
@@ -38,109 +48,120 @@ defaultProject =
           , projectCarpSearchPaths = []
           , projectPrintTypedAST = False
           , projectCompiler = case platform of
-                                Windows -> "clang-cl.exe -D _CRT_SECURE_NO_WARNINGS"
-                                _ ->       "clang -fPIC -lm -g"
+                                Windows -> "clang-cl.exe"
+                                _ ->       "clang"
           , projectCore = True
           , projectEchoCompilationCommand = False
           , projectCanExecute = False
           , projectFilePathPrintLength = FullPath
           , projectGenerateOnly = False
           , projectForceReload = False
           }
+  where win = platform == Windows
```

Comment:
Looks like this local binding isn't used?

Reply:
Oh, right, thanks.